### ISSUE TITLE
Thinner borders with WQ timer?

### ISSUE THREAD

**Issue by MaximusPM** [p3lim-wow/BetterWorldQuests#46]
Is it possible to wither make the borders display the remaining time on that WQ, like World Quest Tab used to do before it was abandoned?

Also, if not, maybe just make the borders a bit thinner?

I really like how this addon is pretty simple, without hundreds of weird features, but I really miss the timer on the borders from WQT.

**Comment by p3lim** [comment]
You mean like the border itself is a circular timer? That sounds cool, although flashy for no real reason, considering how world quests work now.

As for thin-ness - the latest version changed to use the same style as the new default map pins, which are literally 1px.

**Comment by MaximusPM** [comment]
> You mean like the border itself is a circular timer?

Exaclty

**Comment by MaximusPM** [comment]
> That sounds cool, although flashy for no real reason

Makes it much easier to see which ones are about to end, which ones still have a long time to go at a quick glance.

**Comment by MaximusPM** [comment]
> As for thin-ness - the latest version changed to use the same style as the new default map pins, which are literally 1px.

Ah, just tested here the new version, so much better

**Comment by conundrum89** [comment]
I would love the ability to see which WQs are expiring "soon" - maybe with "soon" being a configurable value. I don't need to see a precise time left or anything, but being able to prioritise ones that are gonna expire before I play next is the only feature I miss from the much much heavier WQ addons!

**Comment by p3lim** [comment]
The default "soon" indicator already exists, I think it shows when it's <1h left.

**Comment by conundrum89** [comment]
Yeah, 1 hour is kinda useless though :( If it were a setting, I'd probably set it to 24 hours, so I know which WQs I should do if I can't play again until the same time tomorrow

**Comment by p3lim** [comment]
That is very fair, and I'll be sure to include it in the rewrite. I'll add two thresholds; one for a yellow icon that shows when its <=24h left, and one for the red icon that shows when it's <=1h left. Both thresholds will be configurable.

**Comment by conundrum89** [comment]
You rock, thankyou!
